exercises/isogram/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Generated by Cargo
+# will have compiled files and executables
+/target/
+/.idea/
```

Comment:
I think .idea should not be here. It is not generated by Cargo, so the comment is inaccurate.

```
$ cargo new testproj; cat testproj/.gitignore
     Created library `testproj` project
/target/
**/*.rs.bk
Cargo.lock
```

I think things other than those three entries should not be present.